app/js/views/themes.jsx

```diff
@@ -11,39 +14,15 @@ export default class Themes extends React.Component {
     };
 
     this.foxbox = props.foxbox;
+    this.update = this.update.bind(this);
   }
 
   componentDidMount() {
-    this.foxbox.getRecipes()
-      .then(themes => {
-        this.setState({ themes });
-      })
-      .catch(console.error.bind(console));
+    this.update();
   }
 
-  /**
-   * Activate or deactivate a recipe.
-   *
-   * @param {number} id
-   * @param {SyntheticEvent} evt
-   */
-  handleOnChange(id, evt) {
-    const value = evt.target.checked;
-
-    this.foxbox.toggleRecipe(id, value)
-      .then(themes => {
-        this.setState({ themes });
-      })
-      .catch(console.error.bind(console));
-  }
-
-  /**
-   * Delete a recipe.
-   *
-   * @param {number} id
-   */
-  handleOnDelete(id) {
-    this.foxbox.removeRecipe(id)
+  update() {
+    this.foxbox.recipes.getAll()
```

Comment:
__nit:__ well we should either call "them" ```recipes``` or ```themes``` - it feels confusing to get themes when I request recipes :)